std/system.d

```diff
@@ -14,7 +14,20 @@ immutable
 {
     /++
         Operating system.
-
+        
+        Example:
+        ----
+        import std.system, std.stdio;
+        int main(string[] args) {
```

Comment:
Please use [Phobos style](http://dlang.org/dstyle.html) for all code examples.

Specifically, all opening braces `{` should be on their own line, and 4-space indentation should be used.